projects/gloo/api/external/envoy/config/trace/v3/opentelemetry.proto

```diff
@@ -43,6 +44,10 @@ message OpenTelemetryConfig {
     // Optional gRPC transport options
     GrpcService grpc_service = 4;
   }
+
+  // Optional. The maximum number of spans that can be cached.
+  // If not specified, the default is 1024.
+  google.protobuf.UInt32Value max_cache_size = 5;
```

Comment:
nit: a little more documentation here would be nice. The main reason we are adding this field is to prevent customers from seeing traces getting dropped. We might want to incorporate this information into the field a bit.